site/active.js

```diff
@@ -0,0 +1,11 @@
+var current = document.getElementById('page').getElementsByTagName('a');
+
+function Active() {
+  for (var i=0; i < current.length; i++) {
+    if (document.location.href == current[i].href) {
+      current[i].classList.add('active');
+    }
```

Comment:
minor quibble, but it's generally considered best practice to use the strict comparison in js (ie `===` or `!==`) this will lead to less headaches down the road.

Otherwise, super neat and succinct function!